bin/inbox-start.py

```diff
@@ -149,7 +146,8 @@ def main(prod, enable_tracer, enable_profiler, config, process_num, exit_after):
     http_frontend = SyncHTTPFrontend(
         sync_service, port, enable_tracer, enable_profiler_api
     )
-    sync_service.register_pending_avgs_provider(http_frontend)
+    if enable_tracer:
+        sync_service.register_pending_avgs_provider(http_frontend)
```

Comment:
This is greenlet specific, if gevent is diabled the tracer is disabled as well and it does not make sense.